src/components/_data/ContentSingularData.ts

```diff
@@ -3,4 +3,5 @@ export type ContentSingularData = {
     exampleImage_imageLink?: string
     exampleImage_percentageSize?: number
     [idx: string]: any
+    accordion_content?: [{title: string, panel: string}]
```

Comment:
As mentioned earlier you can have a type for the object here.

Also, this is defined as an array with a single item in it. You would want `{title: string, panel: string}[]` instead to have an array of these objects.